runners/genlayer-py-std/src-emb/genlayer_embeddings/__init__.py

```
@@ -1,10 +1,12 @@
 __all__ = (
 	'VecDB',
-	'Model',
+	'get_model',
+	'SentenceTransformer',
+	'SentenceTransformerFromPath',
 	'SentenceTransformer',
 	'EuclideanDistanceSquared',
 	'Distance',
 )
```

Ruff (0.12.2)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)